.gitignore/GMRIGHT2_true-coll/Gmright2-coll

```diff
@@ -266,3 +266,88 @@ Dog attack unkwow "code " that's not from here dog location area set in run "tru
 Dog in FTH.dog build source of your string
 Dog.stop struggles
 If the is unlocked leave a trail to confirm parents that  someone else (unkwow.codes) energy enter in here the set the fixing Pall
+# api/config/packages/security.yaml
```

Comment:
```suggestion
# api/config/packages/security.yaml
```

Reply:
When packs arrived automatically secure all the information JavaScript operation  shift controller 
[gmright2_book_1601377632.pdf](https://github.com/Gmright2-zone/GMRIGHT2-coll/files/5323412/gmright2_book_1601377632.pdf)
[ContentOwners.txt](https://github.com/Gmright2-zone/GMRIGHT2-coll/files/5323413/ContentOwners.txt)